src/components/MenuItem/MenuItem.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./MenuItem.scss";
+
+export const MenuItem = (props) => {
+    return (
+        <div className="MenuItem">
+            <div className="crossIcon">
+                <props.IconElement/>
```

Comment:
This is also wrong. We created the `<Icon/>` component for a reason. So we could use that component instead of manualy rendering the svg every time. We did this because it will allow us to control the size and colour of svg icons without having to write custom css every time. Right now you dont notice that because the icons are exported at the default size we want em at and the default color we want em at, but this code falls over. 

Second: You do not need to pass a custom icon to this component. Why? Because we know that the menu Item component will always use the Circle Cross icon, why would it ever use anythign else? Remember components are about reusability, the attributes expose things that should be different upon each reuse, things that will always be the same (the icon) should be hard coded.

You need to redo this part the correct way. Use the `<Icon>` comp. 

P.S im puposly giving you little info on how the correc thing looks like.